### PR TITLE
Closes #156 - Change ADAM to ADaM; remove date

### LINKED ISSUE #156: Documentation:  update `Explore ADaM Datasets` vignette
### Please select a category the issue is focused on?

User Guides

### Let us know where something needs a refresh or put your idea here!

- [x] Update the title as `Explore ADaM Datasets`
- [x] Remove the date

Image

### PR DESCRIPTION
**Pull request by jimrothstein**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build pharmaverseadam site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new datasets occur on the "[Reference](https://pharmaverse.github.io/pharmaverseadam/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: vignettes/articles/preview-adam.Rmd [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
-title: "Explore ADAM Datasets"
-date: "2026-01-17"
+title: "Explore ADaM Datasets"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{Explore ADAM Datasets}
```
Comment (Fanny-Gautier) [inline vignettes/articles/preview-adam.Rmd]: Ensure no warning while building the site and align Title with the navigation bar.

Image

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]
Please address the below comment and complete the tasks of the PR checklist.

File: vignettes/articles/preview-adam.Rmd (hunk 1 above)
Comment (jimrothstein) [reply vignettes/articles/preview-adam.Rmd]: "align Title with the navigation bar"

I am not sure what you mean here.
vin
vin

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]
Please update the navigation bar accordingly,

Image

and complete the tasks from the PR checklist, including update of `NEWS.md`

**Comment by jimrothstein** [comment]
@Fanny-Gautier 
Update News.md?
Add for #102  "by theraputic area" ??

news

**Comment by Fanny-Gautier** [comment]
> @Fanny-Gautier Update News.md? Add for #102 "by theraputic area" ??
> 
> news

@jimrothstein Please update `NEWS.md` within `pharmaverseadam (development version)` and subsection `Developer Notes` with the modification applied (e.g. Updated title of the vignette `Explore ADaM datasets`. #... ) ....
Please also update the title of this PR as per the checklist: `Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)`
Please let me know when this PR could be completed, `{pharmaverseadam}` release is expected soon. 
Thank you.

**Comment by jimrothstein** [comment]
@Fanny-Gautier 

Made changes.
(In future, if any small changes come up, please do ask.   I would like to stay in loop.  Thx)

[x] 1. update NEWS.md within pharmaverseadam (development version) and subsection Developer Notes with the modification applied (e.g. Updated title of the vignette Explore ADaM datasets. #... ) ....

[x] 2. 
Please also update the title of this PR as per the checklist: Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)

[x] 3. Please let me know when this PR could be completed, {pharmaverseadam} release is expected soon.

**Review by jimrothstein — COMMENTED** [review]
Attempt to sink positron and github, branch #156

**Comment by jimrothstein** [comment]
@Fanny-Gautier 
Not sure why 2 failures.
Locally, did 'git pull' ; check() - no errors.

**Comment by jimrothstein** [comment]
@Fanny-Gautier 

Latest error is in `common.yml`, but I am hestitant go further.    Here is what `copilot` is saying:

```
Job failure root cause

    The job failed because the lycheeverse/lychee-action (the "links" check) exited with a non-zero status (Process completed with exit code 2). The action was configured to fail the job on link problems (inputs shown in the logs: fail: true, jobSummary: true, format: markdown).
    The log excerpt shows lychee followed a CRAN redirect to a valid page (200), but the action still returned exit code 2 — indicating at least one checked link failed (or lychee returned a non-zero exit status for another reason). The workflow currently uses the reusable links workflow from pharmaverse/admiralci, so the failure originates from that lychee run.

Immediate, low-risk fixes

    Stop the links job from failing the whole CI while preserving the report
        Change lychee’s fail behaviour to false so link problems generate a summary but do not fail the job. This is the quickest way to restore green CI while you investigate the specific broken links shown in the job summary.
        Implement this by replacing the reusable links job with a local job that calls lychee and sets fail: false, or by modifying the upstream reusable workflow to change the default.

Example: replace the "links" job in .github/workflows/common.yml with a local job that runs lychee but doesn’t fail the workflow

Replace the existing block: links: name: Links uses: pharmaverse/admiralci/.github/workflows/links.yml@main if: github.event_name == 'pull_request'

With this local job (add the steps into .github/workflows/common.yml at the same location):

links: name: Links runs-on: ubuntu-latest if: github.event_name == 'pull_request' needs: get_r_version steps: - name: Checkout uses: actions/checkout@v4
Code

- name: Run lychee link check (non-fatal)
  uses: lycheeverse/lychee-action@v2
  with:
    fail: false
    format: markdown
    jobSummary: true

Notes:

    This will keep the lychee report in the job summary and logs but will not make the whole CI fail due to link issues.
    Use this as a temporary remediation so other PR checks are not blocked.

Recommended long-term fixes 2) Fix the underlying link(s)

    Open the job summary (the workflow run’s summary link printed in the logs) to see the complete list of failing links and fix the offending URLs in the repository (docs, README, pkgdown site content, vignettes, etc.). The logs provided a summary link earlier: https://github.com/pharmaverse/pharmaverseadam/actions/runs/28477825506#summary-84406778581

    Harden the links job behavior upstream
        In pharmaverse/admiralci/.github/workflows/links.yml (the reusable workflow), add:
            retries or backoff for transient HTTP failures (some link checks are flaky).
            filtering/ignore list for known redirects that are benign (for example, CRAN redirects).
            keep jobSummary enabled and consider failing only on a threshold of broken links instead of any single failure.
        If you maintain the upstream reusable workflow, change the default lychee input fail to false or add an input to allow callers to select fail behavior. Example input in the reusable workflow: inputs: lychee_fail: description: 'Whether to fail the job for link errors' required: false default: 'true' Then use it when calling lychee: uses: lycheeverse/lychee-action@v2 with: fail: ${{ inputs.lychee_fail }}

    Re-run after fixes
        After either (A) making lychee non-fatal or (B) fixing the broken links, re-run the workflow to confirm the problem is resolved.

Why I recommend the local override first

    The reusable workflow is maintained in another repo; editing it requires a PR there and takes longer. Replacing the links job locally lets you get green CI quickly while you triage and permanently fix the broken link(s).
```

File: NEWS.md [hunk 2]
```diff
@@ -28,6 +28,8 @@
 
 - Added downloads badge to the website. (#151)
 
+- Updated title of the vignette Explore ADaM datasets. (ADAM -> ADaM) (#156)
```
Comment (Fanny-Gautier) [inline NEWS.md]:
```suggestion
- Updated title of the vignette Explore ADaM datasets (ADAM -> ADaM). (#156) 
```

Comment (Fanny-Gautier) [reply NEWS.md]: This must go under `# pharmaverseadam (development version)` section, not within the CRAN released one.

File: NEWS.md [hunk 3]
```diff
@@ -67,6 +69,8 @@
 
 - Added copyright holder logos. (#113)
 
+- Updated Explore ADaM Dataset Vignette (ADAM -> ADaM) (#156)
```
Comment (Fanny-Gautier) [inline NEWS.md]: No, only the development version has been updated. We do not and cannot modify versions which have already been submitted to CRAN.

File: README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 [![pharmaverse adam Badge](http://pharmaverse.org/shields/pharmaverseadam.svg)](https://pharmaverse.org)
 [![CRAN status](https://www.r-pkg.org/badges/version/pharmaverseadam)](https://CRAN.R-project.org/package=pharmaverseadam)
-[![Total Downloads](http://cranlogs.r-pkg.org/badges/grand-total/pharmaverseadam?color=green)](https://cran.r-project.org/package=pharmaverseadam)
+[![Total Downloads](http://cranlogs.r-pkg.org/badges/grand-total/pharmaverseadam?color=green)](https://cran.r-project.org/web/packages/pharmaverseadam/index.html)
```
Comment (Fanny-Gautier) [inline README.md]: Could you please clarify the reason of this update?

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]

**Comment by Fanny-Gautier** [comment]
The CI/CD `Links / Validate Links` check failure has been fixed into #160. It will be merge to main branch once it has been approved. You will need to merge main to the branch 156 once previous merge has been performed.

File: README.md (hunk 4 above)
Comment (jimrothstein) [reply README.md]: @Fanny-Gautier 
The github action that is triggered on push did not like the url in original form.  Copilot said I was using wrong 'canonical' form and suggested this change, which I did, and this fixed the url error.

File: NEWS.md (hunk 3 above)
Comment (jimrothstein) [reply NEWS.md]: @Fanny-Gautier

Just to be sure, I should add to top of News.md
<hr>

# pharmaverseadam (development version)

## New Features

## Documentation



<details>
<summary>Developer Notes</summary>

- Updated title of the vignette Explore ADaM datasets. (ADAM -> ADaM) (#156)

</details>
<hr>

File: README.md (hunk 4 above)
Comment (Fanny-Gautier) [reply README.md]: I do not face this issue with other PRs. Could you please provide a screenshot of the error? Thank you.

File: NEWS.md (hunk 3 above)
Comment (Fanny-Gautier) [reply NEWS.md]: No need of `## New Fetaures` nor `## Documentation`. Only Developer Notes is necessary. Thank you.

File: README.md (hunk 4 above)
Comment (jimrothstein) [reply README.md]: @Fanny-Gautier 
```
With the URL returned to orginal way, there is an error:

This is what "explain error" (copilot) reponses with:

```

I'll help you find a solution for this failing job. Let me first get the job logs.Let me get the workflow file to understand the job configuration better:## Solution: Link Check Failure

The job is failing due to a **broken link detection** in the `links` workflow (Lychee link checker). The error shows:

```
[200] <https://cran.r-project.org/package=pharmaverseadam> | Redirect: Followed 1 redirect...
Process completed with exit code 2.
```

**The Issue:**
The link checker is configured with `fail: true`, which means it exits with code 2 when it encounters redirects. The CRAN link is returning a 303 redirect, which is valid but causing the job to fail.

**Solutions:**

1. **Update the Lychee configuration** to allow redirects (recommended):
   - Create or update `.lychee.toml` in your repository root with:
   ```toml
   [http]
   allow_redirects = true
   ```

2. **Relax the link checker in the workflow** by modifying the `links` job in `.github/workflows/common.yml`:
   ```yaml
   links:
     name: Links
     uses: pharmaverse/admiralci/.github/workflows/links.yml@main
     if: github.event_name == 'pull_request'
     with:
       fail-on-redirect: false  # If this parameter is supported
   ```

3. **Update the documentation URL** to use the direct target after redirect:
   - Change `https://cran.r-project.org/package=pharmaverseadam` to `https://cran.r-project.org/web/packages/pharmaverseadam/index.html`

**Recommended Action:** Implement option 1 by creating a `.lychee.toml` configuration file that allows redirects, as CRAN redirects are expected and valid.

Comment (jimrothstein) [reply README.md]: Screenshot   (from github actions )
url

File: NEWS.md (hunk 3 above)
Comment (jimrothstein) [reply NEWS.md]: Not trying to be dense.  I do not know WHERE to put the line 
`- Updated title of the vignette Explore ADaM datasets. (ADAM -> ADaM) (#156) `

I know you do not want at current location.

- I am using branch:  `156_Documention_fix_title`
- Below is structure of NEWS.md, including the line in question (in `wrong` location)
- 
- Where in the structure do you want this line?


```
# pharmaverseadam 1.3.0

## New features
...
...

## Documentation

...
...

<details>
<summary>Developer Notes</summary>

...
...
...

- Added downloads badge to the website. (#151)

- Updated title of the vignette Explore ADaM datasets. (ADAM -> ADaM) (#156) 

</details>

# pharmaverseadam 1.2.0
...
...
...
...

```

File: README.md (hunk 4 above)
Comment (Fanny-Gautier) [reply README.md]: Hi @jimrothstein ,
The error comes from the `pull_request_template.md` that I fixed in #160.
image
There is no error with `README.md`.

File: NEWS.md (hunk 3 above)
Comment (Fanny-Gautier) [reply NEWS.md]:
```
# pharmaverseadam (development version)

<details>
<summary>Developer Notes</summary>

- Updated title of the vignette Explore ADaM datasets. (#156) 

</details>
```

**Comment by jimrothstein** [comment]
@Fanny-Gautier 

This may too much, perhaps  structured documents like NEWS.md could be explained as below.

Then you could have said,  "add a new version at top ... and include "developer_notes" section.
```
{
  "version": "pharmaverseadam 1.3.0",
  "sections": [
    {
      "heading": "New features",
      "items": [
        "Added `ADAPET`, `ADTPET` and `ADNV` from `{admiralneuro}` using package version 0.2.1. (#138)"
        // ... 0, 1, or n additional items
      ],
      "developer_notes": []
    },
    {
      "heading": "Documentation",
      "items": [
        "Refreshed `{pharmaverseadam}` datasets using `{admiral}` 1.4.1, ..."
        // ... 0, 1, or n additional items
      ],
      "developer_notes": [
        "Updated `{lintr}` configurations ... (#133)"
        // ... 0, 1, or n additional items
      ]
    }
  ]
}
```

File: NEWS.md [hunk 5]
```diff
@@ -1,3 +1,13 @@
+
```
Comment (Fanny-Gautier) [inline NEWS.md]: Please delete extra blank line from the top.

File: NEWS.md
Comment (Fanny-Gautier) [inline NEWS.md]: Please delete this line 80, it hasn't been updated in version 1.2.0.

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]
Please, place the update `NEWS.md` within `pharmaverseadam (development version)` section only.

**Comment by Fanny-Gautier** [comment]
> @Fanny-Gautier
> 
> This may too much, perhaps structured documents like NEWS.md could be explained as below.
> 
> Then you could have said, "add a new version at top ... and include "developer_notes" section.
> 
> ```
> {
>   "version": "pharmaverseadam 1.3.0",
>   "sections": [
>     {
>       "heading": "New features",
>       "items": [
>         "Added `ADAPET`, `ADTPET` and `ADNV` from `{admiralneuro}` using package version 0.2.1. (#138)"
>         // ... 0, 1, or n additional items
>       ],
>       "developer_notes": []
>     },
>     {
>       "heading": "Documentation",
>       "items": [
>         "Refreshed `{pharmaverseadam}` datasets using `{admiral}` 1.4.1, ..."
>         // ... 0, 1, or n additional items
>       ],
>       "developer_notes": [
>         "Updated `{lintr}` configurations ... (#133)"
>         // ... 0, 1, or n additional items
>       ]
>     }
>   ]
> }
> ```

You can review the R package development [guidelines](https://r-pkgs.org/preface.html), particularly the [`NEWS.md`](https://r-pkgs.org/other-markdown.html#sec-news) section.

**Comment by jimrothstein** [comment]
@Fanny-Gautier 

Apologies Fanny.
Obviously I am not on top of this one. :sad:

Would you mind making the change, then I know what you want.

**Comment by Fanny-Gautier** [comment]
> @Fanny-Gautier
> 
> Apologies Fanny. Obviously I am not on top of this one. :sad:
> 
> Would you mind making the change, then I know what you want.

@jimrothstein, You are almost there! You just need to delete the first extra blank line and row 80, then it will be ready to merge! Don't give up when you're so close 😉

**Comment by jimrothstein** [comment]
@Fanny-Gautier 
Hope better !

**Review by Fanny-Gautier — APPROVED** [review]
This is perfect now! Thank you!

**Comment by jimrothstein** [comment]
@Fanny-Gautier 

I think the _actual_ NEWS.md structure could be useful .  

I was really confused when you said to add to `development version` 
and not 
`pharmaverseadam 1.3.0`.   

There was no `pharmaverseadam (development version)` at that time.

(Now I know.)

```
pharmaverseadam (development version):
  Developer Notes:

pharmaverseadam 1.3.0:
  New features:
  Documentation:
  Developer Notes:

pharmaverseadam 1.2.0:
  New features:
  Documentation:
  Developer Notes:
```
